.github/workflows/ci_tests.yml

```
@@ -14,6 +14,90 @@ jobs:
     - name: Use Node.js
       uses: actions/setup-node@v2
       with:
-        node-version: '15'
+        node-version: '14'
 
-    - run: echo "Workflow is added!"
+    - name: Prepare yarn
+      run: |
+        sudo npm i -g yarn@1.22.4 --force
+        yarn --no-progress
+
+    - name: Build
+      run: |
+        yarn build
+        chmod -R 777 .
+
+    - name: Linter & Jest Tests
+      run: |
+        yarn lint
+        yarn test:ci
+
+    - name: Zip artifacts 
+      run: |
+        7z a -tzip -mx3 -mmt2 artifacts.zip packages
```

Since we execute single-line command, you can use the following:
```suggestion
      run: 7z a -tzip -mx3 -mmt2 artifacts.zip packages
```